StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetAppearance+Helpers.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  PaymentSheetAppearance+Helpers.swift
+//  StripePaymentSheet
+//
+//  Created by Yuki Tokuhiro on 9/15/25.
+//
+
+@_spi(STP) import StripeUICore
+import UIKit
+
+internal extension PaymentSheet.Appearance {
+    var topFormInsets: NSDirectionalEdgeInsets {
+        return .insets(top: formInsets.top, leading: formInsets.leading, trailing: formInsets.trailing)
+    }
+}
+
+extension PaymentSheet.Appearance.NavigationBarStyle {
+    var isGlass: Bool {
+        #if !os(visionOS)
+        guard #available(iOS 26.0, *) else {
+            return false
+        }
+        return self == .glass
+        #else
+        return false
+        #endif
+    }
+    var isPlain: Bool {
+        return self == .plain
+    }
+}
+
+extension UIView {
+    /// A convenience method that looks at `appearance.cornerRadius` and applies it or, if `nil`, handles what the default value should be.
+    /// - Parameter shouldUsePrimaryButtonCornerRadius: If `true`, `appearance.primaryButton.cornerRadius` (if non-nil) takes precedence over `appearance.cornerRadius`.
+    func applyCornerRadius(appearance: PaymentSheet.Appearance, ios26DefaultCornerStyle: CornerStyle = .uniform, shouldUsePrimaryButtonCornerRadius: Bool = false) {
```

Comment:
Seems like we could just have a funciton like:
```
    func resolveCornerRadius(appearance: PaymentSheet.Appearance, shouldUsePrimaryButtonCornerRadius: Bool = false) -> CGFloat? {
        if shouldUsePrimaryButtonCornerRadius, let primaryButtonCornerRadius = appearance.primaryButton.cornerRadius {
            return primaryButtonCornerRadius
        }
        return nil
    }
```

Then call the suggested `applyCornerRadius(cornerRadius: CGFloat, ios26DefaultCornerStyle)` function that i proposed in another comment?

Reply:
chatted offline, ignore this comment.